src/logic/safe/store/models/safe.ts

```diff
@@ -24,6 +24,7 @@ export type SpendingLimit = {
 
 export type SafeRecordProps = {
   address: string
+  chainId: ETHEREUM_NETWORK | ''
```

Comment:
Why can it also be an empty string? Can't you default it to match the `DEFAULT_NETWORK`? It's probably still called `DEFAULT_NETWORK_ID` in this branch.